composer.json

```diff
@@ -22,7 +22,7 @@
     "symfony/yaml": "^2.0.5 || ^3.0 || ^4.0 || ^5.0.0"
   },
   "require-dev": {
-    "phpunit/phpunit": "^8.5",
+    "phpunit/phpunit": "^9.3",
```

Comment:
this makes composer stop on php 7.2 since the minimum version for phpunit there is 7.3
can you let it open to both versions? php 7.2 still receives security updates